rfc_template.md

```diff
@@ -0,0 +1,18 @@
+# 0001: RFC_NAME
```

Comment:
Do we really need numbered RFCs? They will have separate lifetimes and later RFCs might be released earlier etc. Also, it might be hard to know what the next free number is. (One might have to look through open and closed issues etc.)

As a different solution, e.g. in the "C# language" repository, they just [use names for their proposals](https://github.com/dotnet/csharplang/tree/master/proposals).

Reply:
Should be clear what the next free number is when you commit (may be different than when you started, of course). Hopefully we are not accepting proposals so fast that there is some kind of race condition. :)

That said, I am not wedded to numbered RFCs. For me, it's simply a matter of how we would like the files organized, so they aren't just in one big arbitrary pile. Possibly directories like C# uses would also be a good approach.